packages/charts/src/components/LineChart/LineChart.stories.tsx

```diff
@@ -147,10 +147,12 @@ const formattedTemperatureData = ((): LineChartSeries[] => {
         newYork.push({
             value: parseInt(obj['New York'], 10),
             timestamp: new Date(obj.date),
+            valueContext: `${Math.round(parseInt(obj['New York'], 10) / parseInt(obj.Austin, 10))}%`,
```

Comment:
If the idea is to get a percentage, aren't we missing the `* 100`?

```suggestion
            valueContext: `${Math.round((parseInt(obj['New York'], 10) / parseInt(obj.Austin, 10)) * 100)}%`,
```

also would be nice if we were using an util for this to keep it dry

```suggestion
            valueContext: `${getPercentageFromNumStrings(obj['New York'], obj.Austin)}%`,
```

we also have to be careful because we might get a 0 as the denominator. I know this is just a story but it will set the example for other devs

Reply:
Can be anything really, updated the stories to make it clear